server/application-server/src/main/resources/application-prod.yml

```
@@ -13,9 +13,26 @@ spring:
+    password: ${POSTFIX_PASSWORD:}
+    properties:
+      mail:
+        transport:
+          protocol: smtp
+        smtp:
+          starttls:
+            enable: true
+        debug: false
+
 hephaestus:
     host-url: ${APPLICATION_HOST_URL}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)